src/components/tasks/card/index.tsx

```diff
@@ -35,6 +35,8 @@ const Card: FC<Props> = ({
   const iconWidth = '25px';
 
   const cardClassNames = [classNames.card];
+  const dates = JSON.stringify(localEndsOn).slice(1, 11);
```

Comment:
Will this result in runtime exception if `localEndsOn` is nullish or an unexpected value?

Reply:
Done!